TodoListService/Controllers/TodoListController.cs

```diff
@@ -50,7 +56,7 @@ public IEnumerable<TodoItem> Get()
         // POST api/todolist
         public void Post(TodoItem todo)
         {
-            if (null != todo && !string.IsNullOrWhiteSpace(todo.Title))
+            if (todo != null && !string.IsNullOrWhiteSpace(todo.Title))
```

Comment:
>todo != null [](start = 16, length = 12)

fun, some like the pattern of having the null first... I recall reading a best practice book 20 years or so ago suggesting that...